packages/editor-ui/src/components/InputPanel.vue

```diff
@@ -358,18 +335,19 @@ export default defineComponent({
 		pane-type="input"
 		data-test-id="ndv-input-panel"
 		@activate-pane="activatePane"
-		@item-hover="$emit('itemHover', $event)"
+		@item-hover="emit('itemHover', $event)"
 		@link-run="onLinkRun"
 		@unlink-run="onUnlinkRun"
 		@run-change="onRunIndexChange"
-		@table-mounted="$emit('tableMounted', $event)"
-		@search="$emit('search', $event)"
+		@table-mounted="emit('tableMounted', $event)"
+		@search="emit('search', $event)"
```

Comment:
nit: I'd extract these 3 into separate methods as well to align with `onLinkRun`, `onUnlinkRun`, etc.